core/src/main/java/io/neow3j/contract/ScriptBuilder.java

```diff
@@ -128,7 +128,7 @@ public ScriptBuilder pushParam(ContractParameter param) {
                 break;
             case HASH160:
             case HASH256:
```

Comment:
Please add a TODO here which points out that we need to handle the case for HASH256 differently, i.e., cast to Hash256 as soon as the class is availbale